Example/Config/Config-common.xcconfig

```diff
@@ -0,0 +1,2 @@
+// IMPORTANT: MiniApp sample host app version is linked to the SDK version defined in the MiniApp.podspec file
+MARKETING_VERSION = $(RMA_SDK_VERSION)$(RMA_DEMO_APP_BUILD_TYPE)
```

Comment:
this replaces the plist value that uses MARKETING_VERSION, as the developer can accidentally  make the plist come back to the default `$(MARKETING_VERSION)` value